src/main/resources/network/brightspots/rcv/config_file_documentation.txt

```diff
@@ -113,10 +113,59 @@ Config file must be valid JSON format. Examples can be found in the test_data fo
     set of configuration parameters that specify the tabulation rules
     the "rules" section contains the following parameters:
 
+      "randomSeed" required when tiebreakMode is random, previousRoundCountsThenRandom, or generatePermutation
+        the integer seed for the application's pseudorandom number generator
+        value: [0..2147483647]
+        if not supplied: null
+
       "tiebreakMode" required
-        how the program should decide which candidate to eliminate when multiple candidates are tied for last place among continuing candidates in a given round
+        how the program should decide which candidate to eliminate when multiple candidates are tied for last place
+        or which candidate to elect first when:
+          1) electionWinnerMode is set to multiSeatAllowOnlyOneWinnerPerRound, and
+          2) multiple candidates exceed the winning threshold in the same round, and
+          3) at least two of those candidates are tied for the highest vote total in that round
         value: "random" | "interactive" | "previousRoundCountsThenRandom" | "previousRoundCountsThenInteractive" | "usePermutationInConfig" | "generatePermutation"
 
+        we use Java 12 java.util.random for randomness in our tiebreak implementations
```

Comment:
Saying "12" makes it seem specific to that version, which we all dearly hope it will never be.
This phrasing hopefully makes it seem more stable:
> "we use java.util.random for randomness in our tiebreak implementations. Compatible
 methods exist for other languages, e.g. https://pypi.org/project/java-random/

Reply:
The .jar file we create is compiled against Java 12, so this statement is accurate.  If one wanted to reproduce our results you would need to do the same, no?  It's not to say we're somehow tied to this implementation in future releases.

Reply:
You bring up an important topic: what sort of reproducability are we trying to achieve?
As I just noted at https://github.com/BrightSpots/rcv/issues/43#issuecomment-509695201
I think the documentation, code and comments should all help clarify and implement the goal of defining and producing a stable standard definition of everything necessary for folks who want to write compatible software that comes up with exactly the same results as this VVSG reference implementation.

So I'm looking for a more stable definition of the PRNG in use than saying it is what Java ships in version 12. I think it has actually be stable for a long time and will remain stable, and we should frame comments to reflect that, and to help those who are writing compatible software in other languages.

Reply:
@nealmcb changed to your wording in PR #370.

Reply:
Gotcha.  That is the notion of reproducibility I tried to facilitate in this PR so I think we are on the same page.  Is there a library you have in mind?  Practically speaking, for this release we are not going to include any code changes like this, but we could further enhance the documentation, and potentially upgrade our implementation in the future.